drivers/input/keyboard/cypress-touchkey.c

```diff
@@ -93,8 +93,6 @@ static int touchkey_keycode[5] = {0,KEY_MENU , KEY_HOME, KEY_BACK, KEY_SEARCH};
 	|| defined (CONFIG_USA_MODEL_SGH_T769) || defined(CONFIG_USA_MODEL_SGH_I577) || defined(CONFIG_CAN_MODEL_SGH_I577R)\
 	|| defined(CONFIG_USA_MODEL_SGH_I757) || defined(CONFIG_CAN_MODEL_SGH_I757M)
 static int touchkey_keycode[5] = {0,KEY_MENU , KEY_HOMEPAGE, KEY_BACK, KEY_SEARCH};
-#elif defined (CONFIG_JPN_MODEL_SC_05D)
```

Comment:
Will pulling the SC-05D lines from this affect the build for that model? I also compile a rom for the device and want to keep things intact. If this will break my Doccomo build I can work on the commit.